tree_for_rbx_flow.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+usage() { echo "$(basename $0) [-d tractoflow/results] [-o output]" 1>&2; exit 1; }
+
+while getopts "d:o:" args; do
+    case "${args}" in
+        d) d=${OPTARG};;
+        o) o=${OPTARG};;
+        *) usage;;
+    esac
+done
+shift $((OPTIND-1))
+
+if [ -z "${d}" ] || [ -z "${o}" ]; then
+    usage
+fi
+
+echo "tractoflow folder: ${d}"
+echo "Output folder: ${o}"
+
+echo "Building tree..."
+cd $d
+for i in *[!{FRF}]; 
+do
+   echo $i
```

Comment:
Please remove this echo

Reply:
Can I keep it? I like to see that all subjects were read correctly... It's "reassuring", ok?

Reply:
I don like this but I think it is ok. Maybe just add a small echo message like: Data processed or something like this. Just to know that it is the subjects used.